tests/cpu/transforms/LookTransform_tests.cpp

```diff
@@ -559,3 +562,161 @@ environment: { FILE1: cdl_test1.cc, FILE2: cdl_test1.cc }
     OCIO_CHECK_EQUAL(std::string("FILE2"), usedContextVars->getStringVarNameByIndex(1));
     OCIO_CHECK_EQUAL(std::string("cdl_test1.cc"), usedContextVars->getStringVarByIndex(1));}
 
+
+OCIO_ADD_TEST(LookTransform, inverse_look_transform)
+{
+    // Test inversion of the transform
```

Comment:
The fix is around a transform containing a look so I suggest `// Test inversion of the transform containing a look.`

Reply:
Done